examples/spatial_demo.py

```
@@ -164,28 +165,22 @@
-spatialRaster.values()
-pyplot.imshow(spatialRaster.values())
-
-# You can also specify the band you want to extract
-spatialRaster.values(band=1)
-
-# You can further modify the source TIF using rasterio
-with rasterio.open(spatialRaster.source) as raster:
+# Open and read the raster using rasterio
+with rasterio.open(spatialRaster["rpath"]) as raster:
     cell_values = raster.read()
 cell_values
```

Ruff (0.15.6)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)